ext/ffi/lib.rs

```diff
@@ -260,6 +260,33 @@ enum NativeType {
   Pointer,
   Buffer,
   Function,
+  Struct(Vec<NativeType>),
+}
+
+impl NativeType {
+  fn get_size(&self) -> usize {
```

Comment:
I think this was discussed earlier as well (maybe on Discord?), but this size calculation isn't correct as it assumes zero padding.

This should instead be implemented using https://doc.rust-lang.org/stable/std/alloc/struct.Layout.html. See this example: https://doc.rust-lang.org/stable/std/alloc/struct.Layout.html#examples

Reply:
https://github.com/denoland/deno/pull/15060#discussion_r929550907

Uh.. @DjDeveloperr, if I may ask, why had those been closed without comment/acknowledgement/implementation?

Reply:
I just marked as resolved because I added padding support to size on my local branch 😅 I'll push soon